recipes/vacumm/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "vacumm" %}
+{% set version = "3.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7107752876047704bd122524bc4a95eaf48ce7bcda6ea8a6e0cda81d1e6be654
+
+build:
+  number: 0
+  #script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win or py3k or np<111]
+
+requirements:
+  build:
+    - libgfortran
+    #- {{ compiler('fortran') }} # not working for unknown reason
```

Comment:
You should use `{{ compiler('fortran') }}` and remove `libgfortran`; the package would otherwise break in the near future. Is there a build log somewhere where that failed?

I *think* this also makes OpenMP work on Mac but I'm not 100% sure; this is something that's going to change in the not-too-distant future. (xref https://github.com/conda-forge/conda-smithy/issues/767#issuecomment-405449391).

Reply:
Here is part of the listing:
````
    gfortran:f90: lib/python/vacumm/misc/grid/interp.f90
    gfortran: error: unrecognized command line option ‘-fno-plt’
    gfortran: error: unrecognized command line option ‘-fno-plt’
    error: Command "/usr/bin/gfortran -Wall -g -fno-second-underscore -fPIC -fopenmp -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -pipe -I/usr/site/tmp/CMO/DEV/sraynaud/Build/builders/conda2/conda-bld/vacumm_1532438998786/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/include -fdebug-prefix-map=${SRC_DIR}=/usr/local/src/conda/${PKG_NAME}-${PKG_VERSION} -fdebug-prefix-map=${PREFIX}=/usr/local/src/conda-prefix -fopenmp -O3 -funroll-loops -Ibuild/src.linux-x86_64-2.7/build/src.linux-x86_64-2.7/vacumm/misc/grid -I/usr/site/tmp/CMO/DEV/sraynaud/Build/builders/conda2/conda-bld/vacumm_1532438998786/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python2.7/site-packages/numpy/core/include -I/usr/site/tmp/CMO/DEV/sraynaud/Build/builders/conda2/conda-bld/vacumm_1532438998786/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/include/python2.7 -c -c lib/python/vacumm/misc/grid/interp.f90 -o build/temp.linux-x86_64-2.7/lib/python/vacumm/misc/grid/interp.o" failed with exit status 1
````    
It seems to use the gfortran of my computer, not the one provided by conda.

Reply:
More of the same log:
````
    compiling Fortran sources
    Fortran f77 compiler: /usr/site/tmp/CMO/DEV/sraynaud/Build/builders/conda2/conda-bld/vacumm_1532438998786/_build_env/bin/x86_64-conda_cos6-linux-gnu-gfortran -Wall -g -ffixed-form -fno-second-underscore -fPIC -fopenmp -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -pipe -I/usr/site/tmp/CMO/DEV/sraynaud/Build/builders/conda2/conda-bld/vacumm_1532438998786/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/include -fdebug-prefix-map=${SRC_DIR}=/usr/local/src/conda/${PKG_NAME}-${PKG_VERSION} -fdebug-prefix-map=${PREFIX}=/usr/local/src/conda-prefix -fopenmp -O3 -funroll-loops
    Fortran f90 compiler: /usr/bin/gfortran -Wall -g -fno-second-underscore -fPIC -fopenmp -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -pipe -I/usr/site/tmp/CMO/DEV/sraynaud/Build/builders/conda2/conda-bld/vacumm_1532438998786/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/include -fdebug-prefix-map=${SRC_DIR}=/usr/local/src/conda/${PKG_NAME}-${PKG_VERSION} -fdebug-prefix-map=${PREFIX}=/usr/local/src/conda-prefix -fopenmp -O3 -funroll-loops
    Fortran fix compiler: /usr/bin/gfortran -Wall -g -ffixed-form -fno-second-underscore -Wall -g -fno-second-underscore -fPIC -fopenmp -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -pipe -I/usr/site/tmp/CMO/DEV/sraynaud/Build/builders/conda2/conda-bld/vacumm_1532438998786/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/include -fdebug-prefix-map=${SRC_DIR}=/usr/local/src/conda/${PKG_NAME}-${PKG_VERSION} -fdebug-prefix-map=${PREFIX}=/usr/local/src/conda-prefix -fopenmp -O3 -funroll-loops
````
Only the f77compiler is taken from conda install.